source/features/bugs-tab.tsx

```diff
@@ -53,7 +54,7 @@ async function init(): Promise<void | false> {
 	const bugsTab = issuesTab.cloneNode(true);
 
 	// Update its appearance
-	select('.octicon', bugsTab)!.replaceWith(bugIcon());
+	select('.octicon', bugsTab)!.replaceWith(<BugIcon/>);
```

Comment:
Downside: if you want consistency (always `BugIcon`, never `bugIcon`), your DOM-returning functions have to use JSX whether you need it or not, or else you'll get:


>  ✖   44:17  A function with a name starting with an uppercase letter should only be used as a constructor.  new-cap

Using JSX silences this error.

Also this means you'll have to `import React from 'dom-chef'` where you normally don't need it.

Reply:
I made changes in https://github.com/vadimdemedes/dom-chef/pull/60 so that plain `<Function/>` calls are as optimized as possible.